.github/workflows/api-integration-tests.yml

```diff
@@ -112,7 +112,6 @@ jobs:
       - name: Functional tests maintenance
         working-directory: ../server
         run: |
-          ./occ migrations:migrate news
           ./occ maintenance:repair
```

Comment:
What is the `maintenance:repair` needed for? Since it is a new and clean setup, nothing should be broken.

Reply:
To make sure repair steps can run succesfully

Reply:
Yea and I think those are only run normally if you do an upgrade, but we don't test upgrades with the CI, I guess we could also drop it here, as it might not catch any bugs.
For that we have the pre-releases

Reply:
I think we should keep it, at least when someone tries to add a new repair step we will see if the php code is working